src/ngraph/frontend/onnx_import/op/resize.cpp

```diff
@@ -0,0 +1,88 @@
+//*****************************************************************************
+// Copyright 2017-2020 Intel Corporation
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//*****************************************************************************
+
+#include "resize.hpp"
+#include "default_opset.hpp"
+#include "exceptions.hpp"
+
+namespace ngraph
+{
+    namespace onnx_import
+    {
+        namespace op
+        {
+            namespace set_1
+            {
+                NodeVector resize(const onnx_import::Node& node)
+                {
+                    const auto inputs = node.get_ng_inputs();
+                    const auto data = inputs.at(0);
+                    const auto scales = inputs.at(1);
+
+                    const auto data_shape = data->get_output_partial_shape(0);
+                    const auto scales_shape = scales->get_output_partial_shape(0);
+
+                    const auto mode = node.get_attribute_value<std::string>("mode", "nearest");
+
+                    std::unordered_set<std::string> supported_modes = {
+                        "nearest", "linear", "cubic", "area"};
```

Comment:
The ONNX spec mentions bilinear, trilinear rather than cubic, area.

Reply:
I listed all modes supported by Interpolate core op, but of course I can shorten this list to
`{"nearest", "linear"}` set with only common modes to the both specifications.